superdesk/publish/transmitters/amazon_sqs_fifo.py

```diff
@@ -30,10 +31,24 @@ class AmazonSQSFIFOPublishService(publish_service.PublishService):
 
     NAME = "Amazon SQS FIFO"
 
-    def _transmit(self, queue_item, subscriber):
-        destination = queue_item.get("destination") or {}
-        config = destination.get("config") or {}
+    def _find_matching_destination(self, subscriber, target_destination):
+        if not subscriber:
+            return {}
+
+        destinations = subscriber.get("destinations") or []
+        delivery_type = (target_destination or {}).get("delivery_type")
+        name = (target_destination or {}).get("name")
+
+        if not delivery_type or not name or not destinations:
+            return {}
+
+        for destination in destinations:
+            if delivery_type and destination.get("name") == name and destination.get("delivery_type"):
```

Comment:
Should the delivery type value be matched, i.e. `delievery_type == destination.get("delivery_type")`?